README.md

```diff
@@ -48,7 +48,7 @@ more. See [the wiki](https://github.com/neovim/neovim/wiki/Installing-Neovim)!
 
 ### Community
```

Comment:
They're more important, they can stay until we need the room for something else.